src/main/scala/tile/FPU.scala

```diff
@@ -835,6 +853,10 @@ class FPU(cfg: FPUParams)(implicit p: Parameters) extends FPUModule()(p) {
     req.fmaCmd := ex_reg_inst(3,2) | (!ex_ctrl.ren3 && ex_reg_inst(27))
     when (ex_cp_valid) {
       req := io.cp_req.bits
+      when (io.cp_req.bits.swap12) {
```

Comment:
why doing this? sharing FPU between vector and scalar?

Reply:
Yes